tls/s2n_early_data.h

```diff
@@ -36,15 +38,15 @@ S2N_RESULT s2n_connection_set_early_data_state(struct s2n_connection *conn, s2n_
 struct s2n_early_data_config {
     uint32_t max_early_data_size;
     uint8_t protocol_version;
-    uint8_t cipher_suite_iana[S2N_TLS_CIPHER_SUITE_LEN];
+    struct s2n_cipher_suite *cipher_suite;
     struct s2n_blob application_protocol;
     struct s2n_blob context;
 };
 S2N_CLEANUP_RESULT s2n_early_data_config_free(struct s2n_early_data_config *config);
+S2N_RESULT s2n_early_data_config_clone(struct s2n_psk *new_psk, struct s2n_early_data_config *old_config);
```

Comment:
Isn't the convention we use in s2n the other way round - old_config first and new_psk  second?
```
S2N_RESULT s2n_early_data_config_clone(struct s2n_early_data_config *old_config, struct s2n_psk *new_psk);
```

Reply:
https://github.com/aws/s2n-tls/pull/2608#discussion_r582188408